data/schema/units/abilities.cfg

```diff
@@ -4,7 +4,21 @@
 		name={NAME}
 		max=infinite
 		super="units/unit_type/attack/specials/" + {NAME}
+        {SIMPLE_KEY female_name t_string}
 		{FILTER_TAG "filter_student" unit ()}
+		{FILTER_TAG "filter_weapon" weapon ()}
+		{FILTER_TAG "filter_second_weapon" weapon ()}
+		{SIMPLE_KEY affect_self bool}
+		{SIMPLE_KEY affect_allies bool}
+		{SIMPLE_KEY affect_enemies bool}
+        [tag]
+		name="affect_adjacent"
+		[key]
+			name="adjacent"
+			type="dir_list"
+		[/key]
+		{FILTER_TAG "filter" unit ()}
+	[/tag]
```

Comment:
I think I'm seeing what the issue here is now - we probably need multiple inheritance here. Try replacing everything from line 6 to 21 with just this line:

```
super="units/unit_type/abilities/~generic~,units/unit_type/attack/specials/" + {NAME}
```